src/alda/repl/commands/ReplCommandManager.java

```diff
@@ -0,0 +1,329 @@
+        "Without arguments, will play the entire score from beginning to end.\n\n" +
+        "Example usage:\n\n" +
+        "  :play\n" +
+        "  :play from 0:05\n" +
+        "  :play to 0:10\n" +
+        "  :play from 0:05 to 0:10\n" +
+        "  :play from guitarIn\n" +
+        "  :play to verse\n" +
+        "  :play from verse to bridge";
+    }
+    @Override
+    public String key() {
+      return "play";
+    }
+  }
+
+  private class ReplSave implements ReplCommand {
+    private String oldSaveFile = null;
+    @Override
+    public void act(String args, StringBuffer history, AldaServer server, ConsoleReader reader) {
+      // Turn ~ into home
+      args = args.replaceFirst("^~",System.getProperty("user.home"));
+      try {
+        if (args.length() == 0) {
+          if (oldSaveFile != null && oldSaveFile.length() != 0) {
+            // Overwrite by default if running :save
+            Files.write(Paths.get(oldSaveFile), history.toString().getBytes());
+          } else {
+            usage();
+          }
+          return;
+        }
+
+        try {
+          Files.write(Paths.get(args), history.toString().getBytes(), StandardOpenOption.CREATE_NEW);
+          oldSaveFile = args;
+        } catch (FileAlreadyExistsException e) {
+          String confirm = reader.readLine("File already present, overwrite? [y/n]: ");
+          if (confirm.equalsIgnoreCase("y") || confirm.equalsIgnoreCase("yes")) {
+            Files.write(Paths.get(args), history.toString().getBytes(), StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING);
+            oldSaveFile = args;
+          }
+        }
+      } catch (IOException|UncheckedIOException e) {
+        e.printStackTrace();
+        System.err.println("There was an error writing to '" + args + "'");
+      }
+    }
+    @Override
+    public String docSummary() {
+      return "Saves an the current REPL session into an Alda score.";
+    }
+    @Override
+    public String docDetails() {
+      return "Usage:\n\n" +
+        "  :save test/examples/bach_cello_suite_no_1.alda\n" +
+        "  :save ~/Scores/love_is_alright_tonite.alda\n\n" +
+        "Once :save has been executed once:\n" +
+        "  :save";
+    }
+    @Override
+    public String key() {
+      return "save";
+    }
+  }
+
+  private class ReplLoad implements ReplCommand {
+    @Override
+    public void act(String args, StringBuffer history, AldaServer server, ConsoleReader reader) {
+
+      if (args == "") {
+        usage();
+        return;
+      }
+      Stream<String> fLines = null;
+      try {
+        fLines = Files.lines(Paths.get(args));
+        StringBuffer newHistory = new StringBuffer();
+        fLines.forEach(x -> {
+            newHistory.append(x);
+            newHistory.append("\n");
+          });
+        history.delete(0, history.length());
+        history.append(newHistory);
+        // TODO check to see if score is valid by sending it to server (maybe with :to 0:00)
```

Comment:
This is a good idea. We could use `alda parse --map` for to parse and evaluate the score without playing it.